test/Atmos/EDMF/closures/pressure.jl

```diff
@@ -0,0 +1,58 @@
+#### Pressure model kernels
+
+"""
+    perturbation_pressure(
+        m::AtmosModel{FT},
+        press::PressureModel,
+        state::Vars,
+        diffusive::Vars,
+        aux::Vars,
+        t::Real,
+        env,
+        i,
+    ) where {FT}
+Returns the value of perturbation pressure gradient
+for updraft i, as well as the corresponding source term
+in the environmental TKE budget following He et al. (JAMES, 2020),
+given:
+ - `m`, an `AtmosModel`
+ - `press`, a `PressureModel`
+ - `state`, state variables
+ - `diffusive`, additional variables
+ - `aux`, auxiliary variables
+ - `t`, the time
+ - `env`, NamedTuple of environment variables
+ - `i`, index of the updraft
+"""
+function perturbation_pressure(
+    m::AtmosModel{FT},
+    press::PressureModel,
+    state::Vars,
+    diffusive::Vars,
+    aux::Vars,
+    t::Real,
+    env,
+    i,
+) where {FT}
+
+    # Alias convention:
+    gm = state
+    en = state.turbconv.environment
+    up = state.turbconv.updraft
+    up_a = aux.turbconv.updraft
```

Comment:
I usually intuitively take `up_a` as updraft area fraction and this caused me a lot confusion when reading thru the computation below. I have to check back and realize it refer to updraft in the aux var. I suggest to change it to a more descriptive varname (I don't mind a longer bit more self-explaining name).

Reply:
Good point. Maybe we should change these to `up_aux`.

Reply:
yes, it is much clearer

Reply:
should we change "en_a" to "en_aux" as well?

Reply:
yes, for consistency

Reply:
Yes, I was planning on starting a slack discussion on alias conventions.